### ISSUE TITLE
Feature: Quick audit answers

### ISSUE THREAD

**Issue by rvantonisse** [AccessibilityNL/accessibility-evaluation-tool#20]
**Is your feature request related to a problem? Please describe.**
As an accessibility auditor I do a lot of audits. When I choose to fail a sample on the audit page, I have to fill in my observations of the violation in a open text field. There are a lot of common failures and every time I have to describe them in the text field while I could easily check them of a list of common failures. When I find a particular violation on multiple pages, I also have to write the same observation under each result per sample. This takes the most of my time while auditing and I am not a fast writer / typist.

**Describe the solution you'd like**
I would like to easily check common failures off a list and provide a description if needed. Example: when I audit a page on non-text alternatives, I want to check the option "Informative image without a text alternative". As description I would only need to write it is about the company's logo.

**Describe alternatives you've considered**
I am already copy pasting from my own most common answers, but that leaves me working with multiple views. I have really general descriptions and just fill in the results for entire sample description field with a list of affected samples. In this way, however, the observations done are not well presented in the jsonld data export from the tool.

**Additional context**
Current tool I am using:
<img width="794" alt="Screenshot of success criterion audit non-text Content in current tool" src="https://user-images.githubusercontent.com/5967550/153028254-9411b285-921a-451a-b158-66a071dea073.png">

New tool:
<img width="587" alt="Screenshot of success criterion audit non-text Content in new tool" src="https://user-images.githubusercontent.com/5967550/153029158-da125f96-401a-4eb4-9a6d-808c8e80dd5e.png">
